indigo_api/models/tasks.py

```diff
@@ -4,7 +4,7 @@
 
 from actstream import action
 from django.core.files.uploadedfile import UploadedFile
-from django.utils.translation import gettext_lazy as __, gettext as _
+from django.utils.translation import ugettext_lazy as __, ugettext as _
```

Comment:
@longhotsummer some other strings are marked with ugettext, instead of ugettext_lazy. Is there a reason?